### PR TITLE
Got rid of sessionJoin and sessionLeave wrappers

### PR DESCRIPTION
**Pull request by or-else**
`ClientComMessage` has a `Session` field now. The wrappers are obsolete now.

### CODE REVIEW

**Comment by or-else** [comment]
Tests pass

File: server/hub.go [hunk 1]
```diff
@@ -29,21 +29,15 @@ var requestLatencyDistribution = []float64{1, 2, 3, 4, 5, 6, 8, 10, 13, 16, 20,
...
-// Request to hub to subscribe session to topic
-type sessionJoin struct {
-	// Message, containing request details.
-	pkt *ClientComMessage
-	// Session to attach to topic.
-	sess *Session
-}
-
+/*
 // Session wants to leave the topic
```
Comment (aforge) [inline server/hub.go:33]: Remove?

File: server/session.go [hunk 2]
```diff
@@ -227,8 +227,8 @@ func (s *Session) unsubAll() {
 
 	for _, sub := range s.subs {
 		// sub.done is the same as topic.unreg
-		// Leave message is not set because the whole session is being dropped.
-		sub.done <- &sessionLeave{sess: s}
+		// Еhe whole session is being dropped.
```
Comment (aforge) [inline server/session.go:230]: s/Ehe/The

**Review by aforge — APPROVED** [review]

File: server/hub.go (hunk 1 above)
Comment (or-else) [reply server/hub.go]: done

File: server/session.go (hunk 2 above)
Comment (or-else) [reply server/session.go]: done

**Pull request merged**